cypress/e2e/StartBuildingYourSoftwareProjectTest.cy.js

```diff
@@ -5,4 +5,9 @@ describe('StartBuildingYourSoftwareProjectTest', () =>{
     it('TC_02.06.001 |Block “Start building your software project” > Verify "Start building your software project” is displayed', ()=>{
 cy.get('h2.h4').should('be.visible');
     });
-});
\ No newline at end of file
+    it('TC_02.06.004 |Block “Start building your software project” >Check redirect to the page "New Job"', ()=>{
+
+        cy.get('a[href="newJob"]').click();
+        cy.url().should('be.eql', 'http://localhost:8080/newJob');
```

Comment:
Please consider to not use http://localhost:8080 in this case.
It would be better to use  `cy.url().should('include','/newJob')`
```suggestion
        cy.url().should('include','/newJob');
```